target/linux/mediatek/image/filogic.mk

```diff
@@ -728,6 +728,24 @@ endif
 endef
 TARGET_DEVICES += xiaomi_redmi-router-ax6000-ubootmod
 
+define Device/yuncore_ax835
+  DEVICE_VENDOR := YunCore
+  DEVICE_MODEL := AX835
+  DEVICE_DTS := mt7981b-yuncore-ax835
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_DTS_LOADADDR := 0x47000000
+  IMAGES := sysupgrade.bin
+  IMAGE_SIZE := 14336k
+  SUPPORTED_DEVICES += mediatek,mt7981-spim-nor-rfb
```

Comment:
This `SUPPORTED_DEVICES += mediatek,mt7981-spim-nor-rfb` line is only needed when you want to support sysupgrade from some OpenWrt installation using the reference board image. Some vendors use this board name. You do not support sysupgrade from the vendor image, please remove this.

Reply:
This is probably needed when you install it using sysupgrade in the vendor firmware.

Reply:
I remembered that I don't have to wait until I get another device.
There still was a flash dump on my computer containing the preinstalled image.